experimental/gittuf/repository_test.go

```diff
@@ -25,9 +23,18 @@ func TestLoadRepository(t *testing.T) {
 	}
 	defer os.Chdir(currentDir) //nolint:errcheck
 
-	repository, err := LoadRepository()
-	assert.Nil(t, err)
-	assert.NotNil(t, repository.r)
+	t.Run("Load with no repo, unsuccessful", func(t *testing.T) {
```

Comment:
I would inline the tmpDir declaration too so that each t.Run has its own tmpDir where a repository is created (or not) with corresponding chdir calls as well. This makes the tests less fragile / dependent on order of execution.